docs/source/redirects.txt

```diff
@@ -4,6 +4,7 @@ get_started/index.rst intro/get_started.rst
 get_started/computers.rst howto/run_codes.rst
 get_started/codes.rst howto/run_codes.rst
 howto/plugins.rst howto/plugins_develop.rst
+howto/exploring.rst howto/query.rst
```

Comment:
Nice! I was about to write a comment about this and then noticed you had already added the redirects. 😄